### PR TITLE
Branch v1.4 adaptable fxml

### PR DESCRIPTION
**Pull request by Robinho98**
UI boxes will now change size to fit the number of filled fields in attraction, itinerary and itinerary attraction

### CODE REVIEW

File: src/main/java/seedu/address/model/itinerary/ItineraryAttraction.java [hunk 1]
```diff
@@ -12,6 +12,7 @@
 public class ItineraryAttraction extends Attraction {
     private final ItineraryTime startTime;
     private final ItineraryTime endTime;
+    private final Attraction attraction;
```
Comment (peter-yeh) [inline src/main/java/seedu/address/model/itinerary/ItineraryAttraction.java:15]: NOOOOOO

File: src/main/java/seedu/address/model/itinerary/ItineraryAttraction.java [hunk 2]
```diff
@@ -47,7 +49,7 @@ public Name getName() {
      * Returns the attraction in this itinerary attraction.
      */
     public Attraction getAttraction() {
-        return this;
+        return attraction;
```
Comment (peter-yeh) [inline src/main/java/seedu/address/model/itinerary/ItineraryAttraction.java]: return super; // if you want the Attraction
return this; // if you want ItineraryAttraction

**Review by peter-yeh — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/itinerary/ItineraryAttraction.java (hunk 2 above)
Comment (Robinho98) [reply src/main/java/seedu/address/model/itinerary/ItineraryAttraction.java]: wait i want the attraction
but it was return this originally

Comment (Robinho98) [reply src/main/java/seedu/address/model/itinerary/ItineraryAttraction.java]: ok nvm i got it haha

Comment (peter-yeh) [reply src/main/java/seedu/address/model/itinerary/ItineraryAttraction.java]: did return super work?

Comment (peter-yeh) [reply src/main/java/seedu/address/model/itinerary/ItineraryAttraction.java]: wait.. why you change back to return this? you shld use super if you need only Attraction

**Review by peter-yeh — APPROVED** [review]
LGTM

**Pull request merged**